src/v2/views/ErrorView/index.tsx

```diff
@@ -31,6 +37,15 @@ function handleRestart() {
 function ErrorView() {
   const { path } = useRouteMatch();
   const [state] = useActor(preloadService);
+  const checkboxRef = useRef<HTMLInputElement>(null);
+
+  const onRestartButtonClick = useCallback(() => {
```

Comment:
```suggestion
  const handleRestartClick = useCallback(() => {
```
Isn't 'handle' correct? cc @BasixKOR

Reply:
It's up to the taste, but I'd prefer `handle` too.